platform/core/README.md

```diff
@@ -28,6 +33,10 @@ Swift bindings), as follows:
 
         cargo build
 
+- Build the Android bindings:
+
+        ./build_for_android.sh
```

Comment:
I don't know if make can't be used. In the other PR we already assume people will have it installed somehow, even in Windows.

Reply:
I have refactored the makefile to also include android build, thus removing the shell script.